enterprise/j2ee.dd/src/org/netbeans/modules/j2ee/dd/api/common/RootInterface.java

```diff
@@ -69,7 +70,7 @@
      *		MERGE_UNION is 	G1 U G2 <=> G1 + E2
      *		MERGE_INTERSECT is	G1 n G2 <=> (G1 U G2) - E1 - E2
      *</pre>
-     * @param bean root of the bean graph that is merged with actual bean graph
+     * @param root root of the bean graph that is merged with actual bean graph
```

Comment:
I think this should have been a delete - the word "bean" and the word "root" in the first position of the documentation are superfluous.

Reply:
Done.